tests/unit/casmutils/coordinate.cpp

```diff
@@ -1,25 +1,66 @@
 #include <casmutils/xtal/coordinate.hpp>
+#include <casmutils/xtal/lattice.hpp>
 #include <gtest/gtest.h>
 #include <memory>
 #include <utility>
 
-class CoordinateTest : public testing::Test
-{
+class CoordinateTest : public testing::Test{
 protected:
-    void SetUp() override
-    {
+    void SetUp() override{
         Eigen::Vector3d raw_coord(0.1, 0.2, 0.3);
+        Eigen::Matrix3d lattice;
+        lattice << 0,0.5,0.5,
+                0.5,0,0.5,
+                0.5,0.5,0;
+        
         coord0_ptr.reset(new rewrap::Coordinate(raw_coord));
```

Comment:
I think that we are going to phase out the ```rewrap``` namespace. Please use ```casmutils::xtal``` to make this phasing out easy.

Reply:
Done